lib/domain-shared/shared_use_cases/account_use_case.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:seeds/datasource/remote/firebase/firebase_message_token_repository.dart';
+
+abstract class AccountUseCase {
+  Future<void> updateFirebaseToken({required String oldAccount, String newAccount = ""}) async {
+    if (oldAccount != newAccount) {
```

Comment:
then this check is 

if(newAccount == null) {
}

Reply:
Ok but why is this blocking the PR??